src/main/java/bank/Account.java

```diff
@@ -201,6 +198,29 @@ private String formatDate(Calendar date, String pattern) {
         Date currentDate = date.getTime();
         return dateFormat.format(currentDate);
     }
+
+    private void addMoney(long value, long accountId) {
+        AccountHistory accountHistory = null;
+        if (accountId == 0) {
+            accountHistory = new AccountHistory(value, AccountHistory.Type.in);
+        } else {
+            AccountHistory history = new AccountHistory(value, AccountHistory.Type.transferIn, accountId);
+        }
+        this.currentBalance += value;
```

Comment:
viết vầy cho gọn: cách tương tự có thể là:
this.currentBalance = this.currentBalance + value;